pyEpiabm/pyEpiabm/tests/test_sweep/test_host_progression_sweep.py

```diff
@@ -38,8 +39,27 @@ def test_update_status(self):
         self.assertRaises(TypeError, test_sweep._update_next_infection_status,
                           self.person2)
 
+    def test_set_latent_time(self):
+        """Tests the set latent time returns a float greater than 0.
```

Comment:
Below time is listed as an int not a float? - think we should be consistent here